ethereum_history_api/circuits/lib/README.md

```diff
@@ -9,14 +18,35 @@ Oracle server is located in [packages/noir-ethereum-api](../packages/noir-ethere
 yarn oracle-server
 ```
 
-After that you can generate proofs by running nargo with `--oracle-resolver`` arg:
+After that you can generate proofs by running nargo with `--oracle-resolver` arg:
 
 ```sh
 nargo prove --oracle-resolver http://localhost:5555
 ```
 
 More details [here](https://noir-lang.org/docs/how_to/how-to-oracles/#step-3---usage-with-nargo)
 
+## Generating verifiers
+
+To generate a Solidity verifier for a package run:
+
+```sh
+nargo codegen-verifier --package package_name
+``` 
+(try `package_name=get_account`).
+
+Be aware that this step may take several minutes when run for the first time. It is not recommended to use VSCode terminal for this command as it seems to run slowly.
```

Comment:
Note: When run for the first time, this step may take up to several minutes. 

Also, it is not recommended to use VSCode terminal for this command as it seems to run slowly.

Reply:
The VsCode terminal does not seem to be an issue. It was an issue with downloading trusted setup